docs/source/roles/system_manager/manage_users.md

```diff
@@ -183,6 +183,18 @@ A sample email might look like the following
 >
 > --details about network and location/VPN restrictions here--
 
+(modifying_network)=
+
+## {{globe_with_meridians}} Manually modifying network restrictions
```

Comment:
_If_ this is going to be under managing users, it might be better to title this something like "Changing access restrictions" to clarify how it relates to users.

Reply:
How about "Changing user network access restrictions", making it clear that this means changing
```suggestion
## {{globe_with_meridians}} Changing user network access restrictions
``` a network setting rather than any other kind of access restriction (such as disabling MFA)